c/zosDiscovery.c

```diff
@@ -523,7 +524,7 @@ int zosDiscoveryServiceInstaller(DataService *dataService, HttpServer *server){
     CrossMemoryServerName *priviligedServerName = getConfiguredProperty(server, HTTP_SERVER_PRIVILEGED_SERVER_PROPERTY);
     state->model = makeZOSModel(priviligedServerName);
     dataService->extension = state;
-    printf("%s end\n", __FUNCTION__);
+    zowelog(NULL, LOG_COMP_ID_MVD_SERVER, ZOWE_LOG_DEBUG, "%s end\n", __FUNCTION__);
```

Comment:
Why is the contents in zosDiscovery.c LOG_COMP_ID_MVD_SERVER?
We have one called LOG_COMP_DISCOVERY
This is related to discovery.c but is the zss API wrapper.
Is it OK to use discovery here? @ifakhrutdinov may have an opinion.

Reply:
@1000TurquoisePogs I have no strong opinion, this was left from Suneeth's best judgement. Personally? It doesn't seem right to me to categorize this with DISCOVERY which is labeled as "Zowe Discovery". I think we should create a new component ID or use the one that handles generic server interaction (ID_MVD_SERVER)

Reply:
That installer function doesn't seem to be directly related to the discovery code, just boiler plate code to establish communication with zss. I would follow the same log component logic as the other installer functions we have.

Reply:
That's fine, it will keep it simple.